wire-ios-data-model/Source/MLS/MLSActionExecutor.swift

```diff
@@ -22,18 +22,94 @@ import WireCoreCrypto
 
 public protocol MLSActionExecutorProtocol {
 
+    /// Processes a welcome message.
+    ///
+    /// - Parameter message: The welcome message to process
+    /// - Returns: The group ID of the group the welcome message was for
+    ///
+    /// If any new CRL distribution points are found, they will be published. They can be observed with ``MLSActionExecutor/onNewCRLsDistributionPoints()``
```

Comment:
Optional - I don't think you have much time: Consider hard wrapping this and other doc comment at 120 lines. They seem to render correctly (still the same) once documentation is built.

Reply:
do you mean 120 characters per line?

Reply:
fixed https://github.com/wireapp/wire-ios/pull/1857/commits/0a6590b72c17249803104e54458a90b00d8c47ce